patches/ReactiveObjC/.patches

```diff
@@ -0,0 +1 @@
+build_conditionally_import_ext_headers_from_framework_or_from.patch
```

Comment:
why do we have a fork _and_ patches? could we settle on one or the other?

Reply:
The fork is for Squirrel.Mac, as in the PR:

> This needs to be a fork commit as it needs a DEPS file to be processed.

We can't patch in a `DEPS` file.

This patch is not for Squirrel.Mac, it is for ReactiveObjC a separate repo.  Here we can use a patch so we're using our existing system.  Squirrel.Mac is just an unfortunate exception

Reply:
we could fold the `DEPS` content from squirrel.mac into `//electron/DEPS`?

Reply:
@nornagon That would be semantically incorrect 🤔  I did it this way so that the DEPS chain actually made sense.  Electron depends on squirrel which depends on these two frameworks

Reply:
maybe, but there's a real benefit to having patches in-repo, which is why we did it for all our dependencies. i don't think it's worth sacrificing real benefit for the abstract concept of "semantic correctness".